### DISCUSSION TITLE
Spec requirements for small project work?

### DISCUSSION THREAD

**Discussion by Relequestual** [json-schema-org/community#212] [Leadership and Teams]
Hey JSON Schema team! (aimed at full time staff, but others are welcome to comment or take part... we need proper teams still.)

If I'm to spec out small bits of work I'd like to see done but don't want to commit to doing myself, how would you like it to be specced out?

- Where should the spec be posted?
- What inforamtion would you want the spec to contain?

These are mostly small operational type projects to augment the productivity of the JSON Schema community!

I thought about posting this on slack, but I also wanted to make sure it could be easily found again.

**Comment by Julian** [comment]
A project board somewhere with links to an issue ticket from whichever repo or repos it applies to would certainly be fine with me. As I'm sure would be a few other possibilities. Basically seems like it would be useful to have a specific collection of issues we can triage and pick up and notify others we're working on, but then have those just be normal things some non-FT person may send a PR for at any point too if they see it before we get to it, which means some sort of issue tickets + board kind of setup.

**Reply by Relequestual** [reply]
I'm less thinking about logistically (I already have plenty of ideas on that) and more about, what sort of content would you want to see in the spec.

I know how to write out a work spec, but doing more in the open arena, and to avoid back and forth, is there any information that should be in a work spec you think should be included that you also think I might not consider? Maybe it doesn't matter and we can figure that out as we go, but maybe you also have some suggestions of things you see people commonly missing.

**Reply by gregsdennis** [reply]
I think an example might be warranted.  I'm having trouble understanding what you're asking for.

**Comment by jviotti** [comment]
Not sure I follow 100%, but based on what you are mentioning, a repository with markdown "spec" files that are merged when approved might be a good start. I know Node.js follows a similar approach
